forem/forem
# Ignore relative url in Images::Optimizer

## What type of PR is this? (check all applicable)

## Description
When we are using our domain-provided asset instead of a CDN, Rails would return them as a relative URL. This updates it so that Optimizer would ignore them and serve it as is. 

## Related Tickets & Documents
Resolves https://github.com/forem/InternalProjectPlanning/issues/154
## QA Instructions, Screenshots, Recordings
1. Enable and start-up Imgproxy. [Please refer to step 1 to step 4 here](https://github.com/forem/forem/pull/10819)
1. Make sure you authenticate your own account either with GitHub or Twitter.
1. Navigate to /new.
1. For the body, enter the following liquid tag `{% user your-own-username %}`
1. Click preview should render the liquid tag correctly, _especially_ those twitter/github icon.
![Screen Shot 2020-10-19 at 3 33 07 PM](https://user-images.githubusercontent.com/15793250/96502873-6bb1cd00-1220-11eb-97b2-9ce936a1564e.png)

## Added tests?

## Added to documentation?

## [optional] Are there any post deployment tasks we need to perform?
